msgcntr/messageforums-app/src/webapp/jsp/discussionForum/statistics/dfStatisticsListByTopic.jsp

```diff
@@ -27,8 +27,8 @@
 %>
 <f:view>
   <sakai:view >
-    <%@ include file="/jsp/discussionForum/menu/forumsMenu.jsp" %>
   	<h:form id="dfStatisticsForm" rendered="#{ForumTool.instructor}">
+		<%@ include file="/jsp/discussionForum/menu/forumsMenu.jsp" %>
```

Comment:
This form has a redered so the tabs won't be visible when then form is not rendered.

Reply:
In my understanding, this should not be a problem.
The same rendered check is used on Forums Main Page (dfFOrums) in the included dfAreaInclude.jsp. and the "Grade" Option. So are not able to visit dfStatistitcsByTopic in the first Place, if you dont have site.upd access permission.
Did I miss something?